apps/src/music/views/PackDialog.module.scss

```diff
@@ -49,63 +49,70 @@
       margin-bottom: 14px;
     }
 
-    .packs {
+    .packsContainer {
       width: 100%;
       height: 100%;
       overflow-y: auto;
       flex: 1;
-      display: grid;
-      grid-template-columns: 25% 25% 25% 25%;
-
       background-color: $neutral_dark;
       border: solid $neutral_dark 2px;
       box-sizing: border-box;
       border-radius: 5px;
+      padding: 16px;
+
+      .packs {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, 232px);
+        grid-gap: 4px;
+        justify-content: space-around;
+
+        .pack {
+          box-sizing: border-box;
+          cursor: pointer;
+          width: 232px;
+          padding: 16px;
+
+          &Selected {
+            background-color: $dark_black;
+            border-radius: 5px;
+          }
 
-      .pack {
-        box-sizing: border-box;
-        padding: 16px;
-        cursor: pointer;
-
-        &Selected {
-          background-color: $dark_black;
-          border-radius: 5px;
-        }
-
-        &Image {
-          width: 100%;
-          transform: scale(1, 1);
-          transition: 0.1s ease-in-out;
-
-          &:hover, &Selected {
-            transform: scale(1.03, 1.03);
+          &Image {
+            width: 100%;
+            transform: scale(1, 1);
             transition: 0.1s ease-in-out;
+            border-radius: 5px;
+
+            &:hover, &Selected {
+              transform: scale(1.03, 1.03);
+              transition: 0.1s ease-in-out;
+            }
           }
-        }
 
-        &Footer {
-          margin-top: 10px;
-          display: flex;
-          justify-content: space-between;
-          align-items: center;
+          &Footer {
+            margin-top: 10px;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
 
-          &Name {
-            font-size: 16px;
-          }
+            &Name {
+              font-size: 16px;
+            }
 
-          &Artist {
-            font-size: 11px;
-          }
+            &Artist {
+              font-size: 11px;
+            }
 
-          &Preview {
-            margin-right: 10px;
+            &Preview {
+              margin-right: 10px;
 
-            .preview {
-              color: $neutral_dark40;
+              .preview {
```

Comment:
Minor - I don't know how strict we've been about this in our codebase but I do think there's a preference to avoid a lot of nesting in CSS files (typically no more than 3 levels deep). So for example I would personally probably pull out the `.preview` class into a top level selector (unless there's a strict need that this rule only apply to `preview` elements inside `.pack` elements)

Reply:
Oh, I didn't know about this.  For these files I've quite enjoyed having the structure of the CSS match the structure of the code.